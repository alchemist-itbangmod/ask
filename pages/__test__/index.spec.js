/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'

import Index from '../index'

describe('With Snapshot Testing', () => {
  it(`[Index] shows "Hello world!"`, () => {
    const component = renderer.create(<Index />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
