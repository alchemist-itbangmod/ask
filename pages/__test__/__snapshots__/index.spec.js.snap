// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing [Index] shows "Hello world!" 1`] = `
<div>
  <div>
    ASK #2
  </div>
</div>
`;
